profile/feature-map.md

```diff
@@ -0,0 +1,323 @@
+
+
+### Supported Credential Formats
+
+| Format        | Description                                 |  
+|:--------------------|:--------------------------------------------|
+| mso_mdoc           |ISO/IEC 18013-5 credential format for verifiable credentials, using a signed Mobile Security Object to bind and protect data elements, with support for selective disclosure.|
+| SD&#8209;JWT&#8209;VC | Credential format developed by the IETF and OpenID Foundation, extending JSON Web Tokens with selective disclosure for web-native verifiable credentials. |
+
+
+## 🧩Features
+
+The following table provides a high-level overview of the key user functionalities supported by the EUDI Wallet Reference Implementation.
+
+| Features        | Description                                 | Status     | 
+|:--------------------|:--------------------------------------------|:----------:|
+| [**Issuance**](#issuance)            | Issuance of verifiable credentials          | In&nbsp;Progress|
+| [**Presentation**](#presentation)        | Presentation of verifiable credentials       | In Progress|
+| [**rQES**](#rqes) | electronic signing with EUDI Wallet    | In Progress |
+| [**Transactions Log**](#transactions-log) | Log of historic transactions executed by the EUDI Wallet         | In Progress|
+| [**Request data deletion to RPs**](#request-data-deletion-to-rps) | Users requesting data deletion from Relying Parties they have previously interacted with     |Planned |
+| [**Report unlawful or suspicious requests to DPA(s)**](#report-unlawful-or-suspicious-requests) | Users reporting suspicious data requests to national Data Protection Authorities   |Planned |
+| [**Attestation Revocation**](#attestation-revocation) | Revoking issued attestations         | Completed |
+| [**Pseudonyms**](#pseudonyms)        | Using pseudonyms (i.e. passkeys) for authentication         | Planned |
+| [**Wallet Revocation**](#wallet-revocation) | Revoking wallet instance         | Planned  |
+| [**Wallet to Wallet interaction**](#wallet-to-wallet) | Enabling the exchange of PID/attestations between two EUDI Wallets       | Planned|
+| [**Wallet Back-Up and Restore**](#wallet-backup-and-restore) | Supporting  procedures for backing up and restoring attestations of an EUDI Wallet       | Planned |
+| [**Migrate to another wallet**](#migrate-to-a-different-wallet) | Supporting  procedures for migrating attestations to another EUDI Wallet      |Planned |
+| [**Support for ZKP**](#support-for-zkp) | Supporting  Zero-Knowledge Proofs in EUDI Wallet interactions      |Planned|
+
+You can explore the [EUDI Wallet Reference Implementation Roadmap](https://github.com/orgs/eu-digital-identity-wallet/projects/24) to explore further details in relation to the features and enhancements that are currently being developed or planned for future implementation.
+</br>
```

Comment:
Replace all </br> and raw <br> with a blank line or <br /> (prefer Markdown blank lines).